awx/main/management/commands/register_default_execution_environments.py

```diff
@@ -84,21 +84,28 @@ def handle(self, *args, **options):
 
             registry_cred.inputs = inputs
             registry_cred.save()
-            changed = True
-
-            if not created:
-                print('Updated Default Execution Environment Credential')
-
-        for ee in reversed(settings.DEFAULT_EXECUTION_ENVIRONMENTS):
-            _, created = ExecutionEnvironment.objects.update_or_create(
-                name=ee["name"], defaults={"image": ee["image"], "managed_by_tower": True, "credential": registry_cred}
-            )
-
-        if created:
-            changed = True
-            print("Default Execution Environment(s) registered.")
+            changed.append(True)
+
+            if not cred_created:
+                print("Updated 'Default Execution Environment Credential'")
+
+        # Create default globally available Execution Environments
+        for ee in reversed(settings.GLOBAL_JOB_EXECUTION_ENVIRONMENTS):
+            _, ee_created = ExecutionEnvironment.objects.update_or_create(name=ee["name"], image=ee["image"], credential=registry_cred)
+            if ee_created:
+                changed.append(True)
+                print(f"'{ee['name']}' Default Execution Environment registered.")
+
+        # Create the control plane execution environment that is used for project updates and system jobs
+        control_plane_ee = settings.CONTROL_PLANE_EXECUTION_ENVIRONMENT
+        _cp_ee, cp_created = ExecutionEnvironment.objects.update_or_create(
+            name="Control Plane Execution Environment", defaults={'image': control_plane_ee, 'managed_by_tower': True, 'credential': registry_cred}
```

Comment:
@AlanCoding you missed this one-- this is the new and ONLY managed_by_tower=True EE that will be used for project updates

Reply:
This may not have been your intent, but I think this would have _jobs_ use the EE named "Control Plane Execution Environment" as their default.

And I like that, we should go with that. What's the harm in jobs using the control plane EE?